medusa/process_tv.py

```diff
@@ -34,650 +35,640 @@
 shutil.copyfile = shutil_custom.copyfile_custom
 
 
-class ProcessResult(object):  # pylint: disable=too-few-public-methods
+class ProcessResult(object):
+
     def __init__(self):
         self.result = True
-        self.output = ''
+        self.output = u''
         self.missedfiles = []
-        self.aggresult = True
+        self.succeeded = True
 
+        self.video_files = []
+        self.rar_content = []
+        self.video_in_rar = []
+        self.not_wanted_files = []
 
-def delete_folder(folder, check_empty=True):
-    """
-    Remove a folder from the filesystem.
+    def _log(self, message, level=logger.INFO):
+        logger.log(message, level)
+        self.output += message + u'\n'
```

Comment:
`self.output.append(message)`
then upon usage of output:
`'\n'.join(self.output)`